cfme/tests/automate/test_simulation.py

```diff
@@ -195,6 +195,7 @@ def test_attribute_value_message(custom_instance):
     """
     Bugzilla:
         1753523
+        1740761
```

Comment:
Is this providing test coverage for 1740761?

Reply:
yes